### ISSUE TITLE
Update YouTube video titles to highlight exciting events.

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#128]
Give each YouTube live meeting a unique title that is indicative of what is discussed. (Preferably a sexy title) You can always rename the hangouts after the recording. One, as a fan, I can't even go back to watch past videos because I don't know what each is about. I'll waste my time going through each one just to find that meeting I liked. (e.g. Meeting when Bd 1.0 or 1.2 launched) More importantly, YouTube is like the second largest search engine and you can be sure Drupalers are searching there. Imagine a Drupaler looking for a video on Views and magically stumbling upon a meeting called "Backdrop CMS: On integrating Drupal Views to core (4/22/13 developer meeting)" By having main topics of discussion reflected on the title, our meetings can appear to the right people. (Who else is searching for stuff like "Views", "content type", etc but one of our target audience?)

Who is going to do this?
* volunteers wanted

Follow-up from https://github.com/backdrop-ops/backdropcms.org/issues/122

**Comment by klonos** [comment]
My time is very limited. I used to download the meetings and watch them at my own pace, but I'm left at the one of February 5 :cry: Sorry.

**Comment by sutibun** [comment]
In the last meeting, the sketchy guy said the meetings pretty much cover the same topics every week with slight variations. Didn't take that into account when suggested optimizing the title.

Hmmm... slight change of plans then.

Use the template and fill in the blanks
-------

Here's what I *now* suggest we do to create the YouTube video titles.

After the live recording ends, from the slight variation the sketchy guy mentioned, think about what was the relatively newest and biggest topic discussed. If you had to choose, what was the highlight topic? Whatever that topic you think of, immediately drop that in the following template:

**Title template:** (where XXX is that topic in your head)

> Backdrop CMS: XXX [Developer meeting MM/DD]

**Description template:** (where a, b, c is a quick and brief list of other topics discussed in the meeting.

> What is Backdrop CMS? Simple. The fork of Drupal 7 that's continuing where Drupal should've gone and providing a true alternative for those not wanting to switch to Drupal 8. But it's more than that. Fully check us out at: https://backdropcms.org/
>
> In this week's CMS meeting, we discuss
>
> - a
> - b
> - c
> - and a whole lot more!

Using last week as an example
--------

For example, in [last weeks meeting](https://www.youtube.com/watch?v=M3VbpaXxT_w) the topic that stood out the most (and I'm biased of course) was the talk on marketing towards the end. Using the template above you'd get the following:

- **YouTube title:** Backdrop CMS: Marketing Backdrop & attracting Drupal users [Developer meeting 12/3]
- **YouTube description:** What is Backdrop CMS? Simple. The fork of Drupal 7 that's continuing where Drupal should've gone and providing a true alternative for those not wanting to switch to Drupal 8. But it's more than that. Fully check us out at: https://backdropcms.org/
   
    In this week's CMS meeting, we discuss
   
    - the first book on Backdrop
    - the security of project browser in both Drupal and Backdrop
    - the working prototype of Drush
    - and a whole lot more!

**I think whoever currently uploads the Youtube video should do this** (or anyone else on the live meeting) Just fill in whatever topic comes to your head. It doesn't need to be perfect. Any custom title would be better than what we have on now.

Since there are 52 weeks in a year, that means we have roughly 52 tries for our custom titles to attract Drupalers. Notice, I purposefully included the word "Drupal" in the description. I did so to boost our video meetings to appear on Drupal related searches.

But if you really want, I can do this. I honestly think it's overkill for someone else to do it. You'll have to wait for me to watch it, submit my title+desc and then someone has to log into Youtube to make the changes. And I don't always watch immediately on Thursdays.

Anyone on the live meeting is the perfect candidate. After the meeting is over, the topics of discussion is still fresh on your minds. You simply plug in as best you can and done. This is no different from coming up with a quick blog post title.

YouTube research and Drupalers
------

As a plus, the monkey did a lil' quick research and here is a small sample of what some Drupal users are searching on YouTube.

![image](https://cloud.githubusercontent.com/assets/14132746/11705645/1952b042-9eb7-11e5-8790-04da541e3cd5.png)

The monkey wasn't sure but as you can see, it's obvious Drupalers are on YouTube. Notice how many search phrases contain "Drupal" ... so, if we could just make sure to mention Drupal in the title every now and then (where it makes sense of course. Let's not force it) we should be fine.

**Comment by jenlampton** [comment]
The YouTube video gets created automatically from the Google Hangout "event". When I create the hangout, i'm asked for a description, where I have started throwing in a link to the agenda (just recently).

After the meeting is over, the event appears on YouTube, and we do have the option to edit the metadata there. I would be willing to turn over control of the YouTube channel to someone if they were willing to update this info after each meeting :)

**Comment by sutibun** [comment]
I'll volunteer if no one else wants to.

Just a few caveats:

- For the most part, I should be able edit within 1-2 days. Probably even that same day. Depending on schedule, may fall behind but no worries, will get to it eventually.
- I will experiment with title formats to see what "clicks". I haven't done YouTube optimization before so this will be a good chance to test things out.
- I will drop the word "Drupal" here and there (where it makes sense). Nothing disparaging. Just name dropping. Don't know if you guys noticed but [one of the weekly videos](https://www.youtube.com/watch?v=nBs4byNK8TY) received 2,786 views. Way above what's normal for the channel. That is the power of these simple changes. We just need to tweak it so it attracts Drupalers...

Basically freedom to play around with the copy.

**Comment by jenlampton** [comment]
We seem to be doing an adequate job with meeting/video names, so closing this issue for now. There's always room for improvement, so feel free to re-open (or open a new issue) if this needs to be readdressed.
